plugins/Login/PasswordResetter.php

```diff
@@ -432,6 +499,28 @@ protected function checkPasswordHash($passwordHash)
         }
     }
 
+    /**
+     * Sends email notification that a password reset process has been cancelled.
+     *
+     * @param array $user User info for the cancelled password reset.
+     */
+    private function sendEmailProcessCancelled(array $user): void
+    {
+        $login = $user['login'];
+        $email = $user['email'];
+
+        $mail = StaticContainer::getContainer()->make(PasswordResetCancelEmail::class, ['login' => $login]);
+        $mail->addTo($email, $login);
+
+        if ($this->emailFromAddress || $this->emailFromName) {
+            $mail->setFrom($this->emailFromAddress, $this->emailFromName);
+        } else {
+            $mail->setDefaultFromPiwik();
+        }
+
+        @$mail->send();
```

Comment:
```suggestion
        $mail->safeSend();
```

That should automatically wrap the sending in a try/catch and log if sending fails for any reason.